src/libsyntax/test.rs

```diff
@@ -297,11 +297,8 @@ fn is_test_fn(cx: &TestCtxt, i: &ast::Item) -> bool {
         match &i.node {
           &ast::ItemFn(ref decl, _, _, ref generics, _) => {
             let no_output = match decl.output {
-                ast::Return(ref ret_ty) => match ret_ty.node {
```

Comment:
Removing this variant seems to have broken [quickcheck](https://github.com/BurntSushi/quickcheck/blob/master/quickcheck_macros/src/lib.rs#L106), because the function returned by [`item_fn`](http://doc.rust-lang.org/syntax/ext/build/trait.AstBuilder.html#tymethod.item_fn) seems to use `ast::Return`, not `ast::DefaultReturn`. Is this something that needs to be fixed on quickcheck's end, or was the switch from `ast::Return(..) => true` to `=> false` unintentional?